proto/api.proto

```diff
@@ -288,15 +277,14 @@ message EndTransactionResponse {
 message RequestUnion {
```

Comment:
Renumbering this proto will break the python client and any third-party clients until their protos are updated. I know we haven't committed to any kind of compatibility yet but since there is also no real benefit to renumbering I would rather leave a gap in the sequence than create unnecessary churn.